local.env

```diff
@@ -10,15 +10,12 @@ WORKDIR=${WORKDIR:=~/mod-workdir}
 
 PLATFORM=$1
 
-if [ -z "${PLATFORM}" ]; then
-  echo "Usage: $0 <platform>"
-  echo "  Where platform can be modduo[-static], modduox[-static], moddwarf or x86_64"
-
-#######################################################################################################################
-
-elif [ "${PLATFORM}" != "modduo" ] && [ "${PLATFORM}" != "modduo-static" ] && [ "${PLATFORM}" != "modduox" ] && [ "${PLATFORM}" != "modduox-static" ] && [ "${PLATFORM}" != "moddwarf" ] && [ "${PLATFORM}" != "x86_64" ]; then
-  echo -e "\e[0;31mPlease provide either modduo, modduox, moddwarf or x86_64 as an argument to this env file\e[0m"
-
+SCRIPT_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+if [[ -z "${PLATFORM}" ]] || [[ ! -a "${SCRIPT_DIR}/toolchain/${PLATFORM}.config" ]]; then
+  echo "Usage: ${BASH_SOURCE[0]} <platform>"
```

Comment:
this assumes the shell is bash, which is not always the case. related to this the previous $0 was also wrong.

Reply:
Just for this file which is sourced, it might be necessary to assume the current directory is mod-plugin-builder

Reply:
that is correct.
I have found some workarounds online, but it gets very ugly. basically trying hacks around all known shells just to find what is the $0 arg when not sourcing the script.

Reply:
Done, in commit c0e8222